### PR TITLE
Feature/kamsiyonna/hero section

### PR DESCRIPTION
**Pull request by KamsiyonnaObi**
@brandonetter Pushed up code for yellow highlight. Any suggestions for making the highlight fit "Software Engineer"?

### CODE REVIEW

**Comment by brandonetter** [comment]
Currently the main page is just a button. Can you update it to show what the style issue you're having is?

**Comment by KamsiyonnaObi** [comment]
> Currently the main page is just a button. Can you update it to show what the style issue you're having is?

The highlight length. (I will create another branch to make a better Home page layout like the one we discussed earlier)

File: components/Hero.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ const Hero = (props: Props) => {
             </p>
             <div className="flex flex-col gap-3.5 font-bold text-[30px] text-black-200">
                 <h1 className="lg:header1 lg:text-[48px] xl:header1">Professional 
-                    <span className="line-clamp-1"> Software Engineer </span> 
-                    Based in Canada
+                    <span className="highlight line-clamp-1"> Software Engineer </span>
```
Comment (brandonetter) [inline components/Hero.tsx]:
```suggestion
                    <span className="highlight line-clamp-1 w-fit"> Software Engineer </span> 
```

**Review by brandonetter — CHANGES_REQUESTED** [review]
Try this

**Comment by brandonetter** [comment]
@KamsiyonnaObi All good? :)

**Comment by KamsiyonnaObi** [comment]
> @KamsiyonnaObi All good? :)
EUREKA!
Screenshot 2023-09-15 at 6 22 30 PM

**Pull request merged**